vnext/Chakra/ChakraExecutor.cpp

```diff
@@ -512,17 +504,20 @@ void ChakraExecutor::loadApplicationScript(
 
   ChakraString jsSourceURL(sourceURL.c_str());
 
+#if defined(OSS_RN)
+  evaluateScript(std::move(script), jsSourceURL);
+#else
+  const ScriptMetadata *scriptMetadata = nullptr;
+  if (m_instanceArgs.ScriptMetadata) {
```

Comment:
>ScriptMetadata [](start = 21, length = 14)

Having the map's value type the same as the variable name is a bit confusing. Would be open to better suggestions.

Reply:
naming looks ok here

---
In reply to: [342380198](https://github.com/microsoft/react-native-windows/pull/3591#discussion_r342380198) [](ancestors = 342380198)